assets/src/styles/blocks/Articles.scss

```diff
@@ -76,6 +76,7 @@
     font-weight: 500;
     color: $grey-80;
   }
+  padding-top: 0 !important;
```

Comment:
Added the `!important` property for a more specificity.
However, I think we should change (not now) the specificity's rules on this file.

i.e. 
```
.article-list-item .article-list-item-headline {}
```
or 
```
.article-list-item {
   .article-list-item-headline {}
}
```
and avoid using `!important`